Added support for Decks by Memrise

EresDev opened the pull request:
Memrise has introduced an alternate Decks to memrise for community based courses. Its support is included in the pull request. More info https://www.memrise.com/decks-by-memrise/

Commit:
Added support for decks.memrise.com

Added support for decks.memrise.com

Commit:
Added info about available support for Decks by Memrise

cooljingle reviewed (approved):
Looks good to me thanks! 
Sorry I've not been around to fix things so appreciate the PR

cooljingle commented:
can you increment the version too?

Commit:
Updated version. 

Minor version update. Almost everyone using this will need this update. It is backward compatible.

EresDev commented:
I have incremented the version. I went for 0.2.0
